digicampipe/io/save_hillas.py

```diff
@@ -34,14 +35,15 @@ def save_hillas_parameters(data_stream, n_showers, output_filename):
         event_number.append(event.r0.event_id)
         time_spread.append(event.dl1.tel[1].time_spread)
         time_stamp.append(event.r0.tel[1].local_camera_clock)
+        border.append(1 if event.dl1.tel[1].on_border else 0)
```

Comment:
maybe tel[1] is not the best here... 
I would suggest to just ask for the telescope_id for this function argument

Also why don't save directly event.dl1.tel[1].on_border ?

the `1 if event.dl1.tel[1].on_border else 0` seems confusing.

Reply:
ok